OpenShift/03_create_cluster.sh

```diff
@@ -76,8 +96,10 @@ mkdir -p ocp
 extract_oc ${OPENSHIFT_RELEASE_IMAGE}
 extract_installer "${OPENSHIFT_RELEASE_IMAGE}" ocp/
 cp install-config.yaml ocp/
+rhcos_image_url
+./gen_metal3_config.sh -u ${RHCOS_IMAGE_URL} > assets/deploy/99-metal3-config-map.yaml
```

Comment:
Can we expose the -i option and allow to override it through a user configured parameter? This is required for VM environments where the nic name is ens3.

Reply:
Would an ENVVariable be good? 



if not , how would you like to see it exposed?

Reply:
Yes, I think an env variable should do the job. 

Another option would be to add it as a var in config_${USER}.sh script.

Reply:
Just thinking, not sure how this is used, but can we just use the bridge - provisioning?

Reply:
afaik the nic name is used for bringing up the static-ip-manager-image container on the master node where we don't have a provisioning bridge:

https://github.com/metal3-io/static-ip-manager-image

Reply:
Thanks - then I script findind the nic name that is used on provisioning NIC on the deploy node, I could use that NIC?

Reply:
The problem is that on the virt environments the nic names are different on the deploy node and on master nodes, I guess it's because the master nodes run RHCOS and the deploy node RHEL. On deploy node the nics are called eth* and on master they're ens*

In any case, if we can override it via an env variable we should be good.